common/src/test/java/org/opensearch/ml/common/transport/model/MLUpdateModelInputTest.java

```diff
@@ -152,7 +154,14 @@ public void readInputStreamSuccessWithNullFields() throws IOException {
     @Test
     public void testToXContent() throws Exception {
         String jsonStr = serializationWithToXContent(updateModelInput);
-        assertEquals(expectedOutputStrForUpdateRequestDoc, jsonStr);
+
+        JSONObject connectorJsonObject = new JSONObject(jsonStr);
+        long connectorCreatedTime = connectorJsonObject.getJSONObject("connector").getLong("created_time");
+        long connectorLastUpdatedTime = connectorJsonObject.getJSONObject("connector").getLong("last_updated_time");
```

Comment:
This is relevant to @ylwu-amzn 's comment about old connectors.  If their value is null, an NPE will happen here: `getLong()` will probably return a null `Long` and then assigning it to a primitive `long` will cause an NPE on unboxing.

Reply:
Good catch, given the new code changes were only new **indexed** connectors will get timestamps in this latest commit c606ffadb77cb044881d9c2d97709ca6d13bce0b this test has been reverted to what it was before as now we aren't changing the the connectors on instantiation the previous test so tests on this previous commit 0d2693180bbd76aab7048490ce9eb4dc5eeb3fe1 (i.e. Before my code changes) will be the same with the addition of two UTs that I said in a previous comment :)